### ISSUE TITLE
Suggested additions for documentation

### ISSUE THREAD

**Issue by don-torok** [OHDSI/WebAPI#120]
Some features/documentation that will help in debugging WebApi connections

 - explicitly define the fields in source and source_daimon and how they are used
 - state that for webapi url's that are target specific, the source_key is the souce_key value in the source table  
 - document how/where to set log (log4j) parameters to make is easier to debug various statements.  For example:
  - the connect statement to the flyway database
  - the connect statement to defined in source.source_connection
  - the SQL string that is sent to the database for a simple test query like getting a vocabulary concept
 
With regard to CDM v4 vs v5:

- Document where to set the value
- If you can have multiple sources, shouldn't you be able to have sources with different CDM versions? Could the CDM version be set in the source table?

**Comment by anthonysena** [comment]
@don-torok unfortunately this issue has not aged well and I'd acknowledge that the points you have raised here are still gaps in the wiki despite our efforts to refresh the content (https://github.com/OHDSI/WebAPI/wiki).

To your points above:

- I think we've addressed your points regarding the source and source_daimon tables in the wiki.
- The `source_key` point is important and should be clearly documented in the code per #1203 in my opinion. Once #1203, the API documentation should be linked from the wiki. 
- Configuring log4j should be covered in the wiki and we can add this to the [sample_settings.xml](https://github.com/OHDSI/WebAPI/blob/master/sample_settings.xml). Connection strings are likely not something logged for security purposes but the hibernate queries can be useful for debugging (although the output can be a bit hard to parse).
- WebAPI only supports CDM v5.x but your point regarding the version makes sense. Something for consideration for WebAPI v3.0.0.
